docs/topics/settings.rst

```diff
@@ -218,6 +218,26 @@ Default: ``None``
 
 The name of the region associated with the AWS client.
 
+.. setting:: ASYNCIO_EVENT_LOOP
+
+ASYNCIO_EVENT_LOOP
+------------------
+
+Default: ``None``
+
+Import path of a given asyncio event loop.
+
+If the asyncio reactor is enabled this setting can be used to specify the 
+asyncio event loop to be used with it. Set the setting to the import path of the 
+desired asyncio event loop class. If the setting is set to ``None`` the default asyncio
+event loop will be used.
+
+You can also specify the event loop path with the function :func:`~scrapy.utils.reactor.install_reactor`:
+
+.. autofunction:: scrapy.utils.reactor.install_reactor
```

Comment:
The reference documentation should only be included once in the documentation, and this is already covered in the documentation of `TWISTED_REACTOR`, so this line should be removed.